LICENSE.txt

```diff
@@ -12,6 +12,10 @@ licensed under the Boost Software License, version 1.0.  This code is found in
 src/mlpack/core/boost_backport/ and more details on the licensing are available
 there.
 
+mlpack contains some usage of the source code of stb, which is licensed 
```

Comment:
Technically, `mlpack may contain some usage of the source of stb (depending on build configuration)`---sometimes the user may have it available on their system.  I know that is pedantic but it is good to be clear. :)

Reply:
I am not sure what I need to add.

Reply:
> I am not sure what I need to add.

@MuLx10 , instead of` mlpack contains` --> `mlpack may contain`